### PR TITLE
Fix link error caused by new layout for SAC installation

### PR DESCRIPTION
**Pull request by core-man**
We only have `linux-install.rst` before #210. Now we split it into `linux-binary.rst` and `linux-source.rst`, so the cross-reference links need updating.

## Before fixing

[ppk 模式下拾取震相](https://seisman.github.io/SAC_Docs_zh/data-process/picking-phase/#id2):

> 该全局变量的设置方式参考 /install/linux-install 一节。

[Environmental variable SACAUX not defined](https://seisman.github.io/SAC_Docs_zh/appendix/messages/msgn/):

> 出现该错误的原因是环境变量 `$SACAUX` 未定义，参考 /install/linux-install。

## After fixing

[ppk 模式下拾取震相](https://sac-docs-zh-1142rnq9g-seisman.vercel.app/data-process/picking-phase/#id2):
> 该全局变量的设置方式参考 Linux 二进制安装 或 Linux 编译安装 一节。

[Environmental variable SACAUX not defined](https://sac-docs-zh-1142rnq9g-seisman.vercel.app/appendix/messages/msgn/):

> 出现该错误的原因是环境变量 `$SACAUX` 未定义，参考 Linux 二进制安装 或 Linux 编译安装。

Addresses #219.

### CODE REVIEW

File: source/appendix/messages/msgn.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Environmental variable SACAUX not defined.
 ------------------------------------------
 
 出现该错误的原因是环境变量 ``$SACAUX`` 未定义，参考
-:doc:`/install/linux-install`\ 。
+:doc:`/install/linux-binary` 或 :doc:`/install/linux-source`\ 。
```
Comment (seisman) [inline source/appendix/messages/msgn.rst]: Should we just link to the 
```
:doc:`/install/index`
```
?

File: source/data-process/picking-phase.rst [hunk 2]
```diff
@@ -61,7 +61,8 @@ SAC 全局变量 ``SAC_PPK_USE_CROSSHAIRS`` 可以控制 ppk 模式下鼠标在
 窗口内的形态。若其值为 ``0``\ ，则鼠标会以十字线的形式出现，即
 “:math:`+`\ ”；当其值为 ``1`` 时，会在十字线的基础上加上水平线
 和垂直线。通常建议设置其值为 ``1``\ ，使得拾取到时时更精确。该全局
-变量的设置方式参考 :doc:`/install/linux-install` 一节。
+变量的设置方式参考 :doc:`/install/linux-binary` 或 :doc:`/install/linux-source`
```
Comment (core-man) [inline source/data-process/picking-phase.rst]: Same for this page as your suggestion in https://github.com/seisman/SAC_Docs_zh/pull/220/files#r642081750 ?

> :doc:`/install/index`

Comment (seisman) [reply source/data-process/picking-phase.rst]: Re-read it again. Maybe not a good idea to link to `install/index`

Comment (core-man) [reply source/data-process/picking-phase.rst]: I've no idea... @seisman for the comment.

File: source/appendix/messages/msgn.rst (hunk 1 above)
Comment (core-man) [reply source/appendix/messages/msgn.rst]: Done in 697138e.

File: source/data-process/picking-phase.rst (hunk 2 above)
Comment (seisman) [reply source/data-process/picking-phase.rst]: Let's link to the `install/index` doc now.

Comment (core-man) [reply source/data-process/picking-phase.rst]: done in 7f4b62c.

**Review by seisman — APPROVED** [review]

**Pull request merged**